otter/integration/tests/test_convergence.py

```diff
@@ -322,7 +410,9 @@ def _choose_random_servers(self, (code, response), n):
         return random.sample(ids, n)
 
     def _delete_those_servers(self, ids, rcs):
-        """Delete each of the servers selected."""
+        """
+        Delete each of the servers selected, and save a list of the
+        ids of the deleted servers."""
```

Comment:
@kivattik That's the job of the TestResources class.  It's a bit underutilized right now because I never had a need for bookkeeping all that data, but if you have a need for it, that's the place to put it.